src/inc/msquic.h

```diff
@@ -925,6 +925,11 @@ typedef struct QUIC_SCHANNEL_CREDENTIAL_ATTRIBUTE_W {
 #define QUIC_PARAM_CONN_ORIG_DEST_CID                   0x05000018  // uint8_t[]
 #define QUIC_PARAM_CONN_SEND_DSCP                       0x05000019  // uint8_t
 
+#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
+#define QUIC_PARAM_CONN_SEND_QTIP                       0x0500001A  // uint8_t (BOOLEAN)
+#define QUIC_PARAM_CONN_RECV_QTIP                       0x0500001B  // uint8_t (BOOLEAN)
+#endif
```

Comment:
```suggestion
#define QUIC_PARAM_CONN_SEND_DSCP                       0x05000019  // uint8_t
#ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
#define QUIC_PARAM_CONN_SEND_QTIP                       0x0500001A  // uint8_t (BOOLEAN)
#define QUIC_PARAM_CONN_RECV_QTIP                       0x0500001B  // uint8_t (BOOLEAN)
#endif
```
As I indicated in the other comment, let's just use one new value `QUIC_PARAM_CONN_QTIP`.